### PR TITLE
[IN-2264] pre-install awscli

### PR DESCRIPTION
**Pull request by Kanan-Bitrise**
### Pull Request Checklist

Check/accept these:

- [x] I've updated the version in the Dockerfile
- [x] I've updated the CHANGELOG.md
- [] I've provided a link or explanation below which describes the changes in the tool itself

`Copy or link the tool's changelog here`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -37,8 +37,13 @@ RUN bitrise stepman update
 # releases: https://github.com/npm/cli/releases
 RUN npm install -g npm@6.13.4
 
-# --- Pre-install jq
-RUN sudo apt-get -y install jq
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install \
+    jq \
+    awscli
+
```
Comment (fehersanyi-bitrise) [inline Dockerfile:44]: why so much white space?

Comment (peterheja-bitrise) [reply Dockerfile:44]: It's a good practice to break it into multiple lines, so in the future we can add more tools here.

**Review by peterheja-bitrise — APPROVED** [review]
LGTM

File: Dockerfile (hunk 1 above)
Comment (fehersanyi-bitrise) [reply Dockerfile:44]: I know, what I meant is that there was like 5 new lines :)

**Review by fehersanyi-bitrise — APPROVED** [review]

**Pull request merged**